lib/still/compiler/incremental/output_to_input_file_registry.ex

```diff
@@ -0,0 +1,39 @@
+defmodule Still.Compiler.Incremental.OutputToInputFileRegistry do
```

Comment:
Can we find a better name for this?

Reply:
Any ideas? I tried to make it descriptive. It's a registry that maps the output file path to its corresponding input file.

Reply:
`IOFileRegistry` sounds good, or even just `FileRegistry`

Reply:
Those sound a bit misleading to me. `FileRegistry` is a bit too generic in this context, don't you think? There are soo many things called `file` that a `FileRegistry` can be anything. I'm not against the `IOFileRegistry`, but I find the letter `IO` to also be very loaded with meaning in the context of application development.